json-form-react/utilities/JsonFormValidator.js

```diff
@@ -55,6 +55,16 @@ export default class JsonFormValidator {
       return this._min(ruleConditions, value);
     case 'max':
       return this._max(ruleConditions, value);
+    case 'letters':
+      return this._validateLetterOnly(value);
+    case 'numbers':
+      return this._validateNumbersOnly(value);
+    case 'email':
+      return this._validateEmail(value);
+    case 'date':
+      return this._validateDate(value);
+    case 'phone':
```

Comment:
@hariidev Can we rename this validation as `phone-lk` so it is clear that this is for Sri Lanka. Because other countries have different phone number formats.